crates/sp-consensus-subspace/src/digests.rs

```diff
@@ -83,6 +94,31 @@ impl Default for PotPreDigest {
     }
 }
 
+impl fmt::Display for PotPreDigest {
```

Comment:
Why not `Debug`? We barely use `Display` for anything more than numbers and this clearly looks like debug implementation to me.